calendar/src/main/java/com/minuk/calendar/PetCalendarView.kt

```diff
@@ -0,0 +1,304 @@
+            val inflater = LayoutInflater.from(parent.context)
+            return PetCalenderViewHolder(ItemCalendarBinding.inflate(inflater))
+        }
+
+        override fun onBindViewHolder(holder: PetCalenderViewHolder, position: Int) {
+
+            val day: Int = getDayFromPosition(calendar[Calendar.MONTH], position).second
+
+            holder.binding.dayCalendarTv.text = "$day"
+
+            //현재 요일의 경우 동그라미 이미지 표시
+            if (position == toDay + startDayOfWeek - 1) {
+                holder.binding.dayCalendarTv.background =
+                    ResourcesCompat.getDrawable(resources, R.drawable.bg_date_selected, null)
+            } else {
+                holder.binding.dayCalendarTv.background = null
+            }
+
+
+            if (position == toDay + startDayOfWeek - 1) {
+                holder.binding.dayCalendarTv.setTextColor(todayColor)
+            } else if (position % 7 == 0 || position % 7 == 6) {
+                holder.binding.dayCalendarTv.setTextColor(weekendColor)
+            } else {
+                holder.binding.dayCalendarTv.setTextColor(weekdaysColor)
+            }
+        }
+
+        override fun getItemCount(): Int {
+            return daysCount
+        }
+
+        /**
+         * position 값으로 날짜 측정하는 함수
+         * 한 달력 화면에서 최대 3개의 월 달력이 보여 월과 일을 구분
+         *
+         * @return month 와 day 를 Pair 객체로 반환합니다.
+         */
+        private fun getDayFromPosition(
+            currentMonth: Int = calendar[Calendar.MONTH],
+            position: Int
+        ): Pair<Int, Int> = when {
+            position < startDayOfWeek -> {
+                val day = endDayOfLastMonth - (startDayOfWeek - position - 1)
+                Pair(currentMonth - 1, day)
+            }
+            position > (endDayOfCurrentMonth + startDayOfWeek - 1) -> {
+                val day = position - (endDayOfCurrentMonth + startDayOfWeek) + 1
+                Pair(currentMonth, day)
+            }
+            else -> {
+                val day = position - startDayOfWeek + 1
+                Pair(currentMonth + 1, day)
+            }
+        }
+
+        private inner class PetCalenderViewHolder(
+            val binding: ItemCalendarBinding
+        ) : RecyclerView.ViewHolder(binding.root) {
+
+            init {
+                initDimensRatio()
+                initEventHandler()
+            }
+
+            private fun initDimensRatio() {
+                val layoutParams =
+                    binding.dayCalendarCl.layoutParams as ConstraintLayout.LayoutParams
+                layoutParams.dimensionRatio = itemViewRatio
+                // Log.d(TAG, itemViewRatio)
+
+                binding.dayCalendarCl.layoutParams = layoutParams
+            }
+
+            private fun initEventHandler() {
+                itemView.setOnClickListener {
+                    val year = calendar[Calendar.YEAR]
+
+                    val pair = getDayFromPosition(position = adapterPosition)
```

Comment:
이렇게 pair를 받아도 되지만
```suggestion
                    val (month, day) = getDayFromPosition(position = adapterPosition)
```
로 받으시면 의미가 명확해집니다

Reply:
오 감사합니다